app/src/main/java/nerd/tuxmobil/fahrplan/congress/preferences/SharedPreferencesRepository.kt

```diff
@@ -35,4 +35,6 @@ interface SharedPreferencesRepository {
     fun getSelectedSessionId(): String
     fun setSelectedSessionId(sessionId: String): Boolean
 
+    fun isScheduleLandscape() : Boolean
```

Comment:
I would prefer a wording like `isForceLandscapeModeEnabled` here an in related variables and functions to avoid misunderstanding. I would omit "schedule" which disallows reusing the preference setting for other screens. More about this in the overall review comment.